### DISCUSSION TITLE
Antenna Definition for a LEO satellite

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#106] [Q&A] [answered]
Dear Developers, 

My colleague and I started processing the kinematic orbits of some LEO satellites, like Sentinel 1A and GRACE Fo, to learn the procedure of doing POD by doing some examples.

For the metadata preparation, I went through the script and the text files for Sentinel 1A, as an example, which you have provided for us on the TUG webpage. When I compare the values in the "010sentinel1A_antennas.txt" with the values for EAST, NORTH and UP in the ANTEX file, they are totally different from each other. To understand it better, I went through the Sentinel 1A documents to see where I can find those values used in the text file, but unfortunately, I couldn't find them. 

Since my purpose is to process the kinematic orbits of another LEO satellite for the purpose of gravity recovery, I would like to make sure to use the correct values for creating the Antenna Definition file. Are L1x, L1y, L1z the same as the NORTH / EAST / UP values in the ANTEX File or I need to find those values somewhere else? 

**From ANTEX file (sen08.atx), we have:**
G01                                                             START OF FREQUENCY  
-11.22     -1.98     59.80                              NORTH / EAST / UP   

**From 010sentinel1A_antennas.txt, we have:**

SENTINEL1A-GPSA   		""      0.001  			-0.0005			0.068  	

Thank you in advance and I am looking forward to hearing from you as soon as possible. 

With Kind Regards, 
Parisa 

[010sentinel1A_antennas.txt](https://github.com/groops-devs/groops/files/9614404/010sentinel1A_antennas.txt)

**Comment by bsuesser-rechberger** [comment, marked as answer]
Dear Parisa,

for our operational solutions, we use the operational values ( in 010sentinel1A_antennas.txt file). But related to the document titled "_Copernicus POD Service: Reprocessing of the Sentinel-1, -2, and -3 orbits_" we will use the recalibrated values (in sen08.atx file) for our reprocessing, which will be done in near future.

Best regards,
Barbara

**Reply by ParisaShafiei** [reply]
Dear Barbara, 

Thank you so much for your reply. Your answer really helped us. Thus, we will use the ANTEX file for the LEO satellite that we want to process its kinematic orbits. 

With Kind Regards,
Parisa
